hiredis-client/ext/redis_client/hiredis/extconf.rb

```diff
@@ -47,7 +47,7 @@ def build_hiredis
 
     env_args = env.map { |k, v| "#{k}=#{v}" }
     Dir.chdir(hiredis_dir) do
-      unless system(make_program, "static", *env_args)
+      unless system(*make_program.split, "static", *env_args)
```

Comment:
I think it would make sense to use `Shellwords.split` instead.

Reply:
good point, updated